content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,199 @@
+  lifecycle management. 
+- When key features are missing, we add them to existing components. For example, we invested in the Job API to cover more use cases like 
+  [IndexedJob](https://kubernetes.io/blog/2021/04/19/introducing-indexed-jobs) and [fixed long standing issues related to pod 
+  tracking](https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers). While this path takes longer to 
+  land features, we believe it is the more sustainable long term solution.
+- Ensuring compatibility with cloud environments where compute resources are elastic and heterogeneous.
+
+For this approach to be feasible, Kueue needs knobs to influence the behavior of those established components so it can effectively manage 
+when and where to start a job. We added those knobs to the `v1.Job` API in the form of two features:
+- [Suspend field](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job), which allows Kueue to signal to the job-controller 
+  when to start or stop a job.
+- [Mutable scheduling directives](https://kubernetes.io/docs/concepts/workloads/controllers/job/#mutable-scheduling-directives), which allows Kueue to 
+  update a job’s nodeSelector before starting it. This way, Kueue can control pod placement while still delegating to kube-scheduler the actual 
+  pod-to-node scheduling.
+
+Note that any custom job API can be managed by Kueue if they offer the above two capabilities.
+
+### The Resource Model
+Kueue defines new APIs to address the requirements mentioned at the beginning of this post. The three main APIs are:
+
+- `ResourceFlavor`: a cluster-scoped API to define resource flavor available for consumption, like a GPU model. At its core, a ResourceFlavor is 
+  a set of labels that mirrors the labels on the nodes that offer those resources.
+- `ClusterQueue`: a cluster-scoped API to define resource pools by setting quotas for one or more resourceFlavor. 
+- `LocalQueue`: a namespaced API for grouping and managing single tenant jobs. In its simplest form, a LocalQueue is a pointer to the ClusterQueue 
+  that the tenant (modeled as a namespace) can use to start their jobs.
+
+For more details, take a look at the [API concepts documentation](https://sigs.k8s.io/kueue/docs/concepts). While the three APIs may look overwhelming, 
+most of Kueue’s operations are centered around ClusterQueue; the ResourceFlavor and LocalQueue APIs are mainly organizational wrappers.
+
+### Example Use Case
+Imagine the following setup for running batch workloads on a Kubernetes cluster on the cloud: 
+- We have two types of autoscaled node groups that differ on their provisioning policies: spot and on-demand. The nodes of each group are 
+  differentiated by the label `instance-type = {spot | ondemand}`. Moreover, since not all jobs can tolerate running on spot nodes, the nodes 
+  are tainted with spot=true:NoSchedule.
```

Comment:
```suggestion
- There are two types of autoscaled node groups that differ on their provisioning policies: spot and on-demand. The nodes of each group are 
  differentiated by the label `"instance-type"="spot"` or `"instance-type"="ondemand"`.
  Moreover, since not all jobs can tolerate running on spot nodes, the nodes are tainted with `"spot"="true:NoSchedule"`.
```

I'm not sure whether to represent the taints and labels as they'd appear in YAML (with a `:`) or JSON (quoted, also with a `:`, or something else. I hope the approach here makes sense to readers.